src/algorithms/optimization/fixed_point_differentiation.jl

```diff
@@ -249,6 +244,38 @@ function _rrule(
     return (env_fixed, info), leading_boundary_fixed_pullback
 end
 
+function _fix_svd_algorithm(alg::SVDAdjoint, signs, info)
+    # embed gauge signs in larger space to fix gauge of full U and V on truncated subspace
+    signs_full = map(zip(signs, space.(info.S_full))) do (σ, S)
```

Comment:
```suggestion
    signs_full = map(zip(signs, info.S_full)) do (σ, S)
```
Not really necessary to broadcast in argument of map: this allocates intermediate object you don't need.